src/main/java/de/wejago/mqtt2influx/service/JsonSubscriber.java

```diff
@@ -65,7 +67,18 @@ private void addMapEntriesByExistingDeviceMappings(Map<String, String> deviceMap
         Map<String, String> messageValueMap = objectMapper.convertValue(entry.getValue(), Map.class);
         for (Map.Entry<String, String> messageValueItem : messageValueMap.entrySet()) {
             if (deviceMappings.containsKey(messageValueItem.getKey())) {
-                deviceToPointProperties.put(deviceMappings.get(messageValueItem.getKey()), messageValueItem.getValue());
+                String value = String.valueOf(messageValueItem.getValue());
```

Comment:
It's not a solution to let ObjectMapper decide on the value type, then force a conversion to String and in the next step convert it to Double.

Just use the solution I suggested:
```
Map<String, String> messageValueMap = objectMapper.convertValue(entry.getValue(), new TypeReference<Map<String, String>>() {
        });
```